packages/sanity/src/core/form/inputs/BooleanInput.tsx

```diff
@@ -46,7 +46,7 @@ export function BooleanInput(props: BooleanInputProps) {
           <FormFieldHeaderText
             description={schemaType.description}
             inputId={id}
-            // validation={validation}
```

Comment:
_Extremely_ curious as to why this was commented out in the first place, but afraid we'll never know

Reply:
😅